src/urllib3/connectionpool.py

```diff
@@ -150,22 +160,22 @@ class HTTPConnectionPool(ConnectionPool, RequestMethods):
     """
 
     scheme = "http"
-    ConnectionCls = HTTPConnection
+    ConnectionCls: Type[Union[HTTPConnection, HTTPSConnection]] = HTTPConnection
     ResponseCls = HTTPResponse
 
     def __init__(
         self,
-        host,
-        port=None,
-        timeout=Timeout.DEFAULT_TIMEOUT,
-        maxsize=1,
-        block=False,
-        headers=None,
-        retries=None,
-        _proxy=None,
-        _proxy_headers=None,
-        _proxy_config=None,
-        **conn_kw,
+        host: Optional[str],
+        port: Optional[int] = None,
+        timeout: object = Timeout.DEFAULT_TIMEOUT,
```

Comment:
Given how often we're doing `timeout` maybe we can add an alias for the type? `_TYPE_TIMEOUT`?

Reply:
Yes, `_TYPE_TIMEOUT` added.